pages/linux/needrestart.md

```diff
@@ -0,0 +1,36 @@
+# needrestart
+
+> Checks which daemons need to be restarted after library upgrades.
+> More information: <https://github.com/liske/needrestart>.
+
+- List outdated processes:
+
+`needrestart`
+
+- Interactively restart services:
+
+`sudo needrestart`
+
+- Run in [v]erbose or [q]uiet mode:
+
+`needrestart -{{v|q}}`
+
+- Check if the kernel is obsolete:
+
+`needrestart -k`
+
+- Check if the CPU microcode is obsolete:
```

Comment:
Is obsolete the correct choice of wording for this use case, I would refer `outdated`

```suggestion
- Check if the CPU microcode is outdated:
```

Reply:
Also if applied here, the command above it uses it.

Reply:
The docs say `obsolete`.

Reply:
Well docs for software isn't always the best.

The definition of obsolete is `No longer in use`, yet in this instance it is still in use

Reply:
Let's use `outdated` then.